settings.gradle

```diff
@@ -192,3 +194,27 @@ if (JavaVersion.current() == JavaVersion.VERSION_1_8) {
   }
 }
 
+dependencyResolutionManagement {
+  versionCatalogs {
+    libs {
+      from(files("./libs.versions.toml"))
+
+      library("delta-standalone", "io.delta:delta-standalone_${scalaVersion}:0.6.0")
+      library("flink115-table-planner", "org.apache.flink:flink-table-planner_${scalaVersion}:1.15.0")
```

Comment:
Hm. The Flink, Jackson, Scala and Spark versions are all declared in `libs.versions.toml`. Having those in two places is going to cause trouble. I doubt that Dependabot is smart enough to update any of these in this block due to the `${scalaVersion}` code block.

Reply:
We already discussed it here.
https://github.com/apache/iceberg/pull/7694#discussion_r1217427024
Let me know if you see a better solution

Reply:
Yes, I've seen the comment and added a couple of suggestions in the other comments.